content/blog/chess-knight-problem/index.md

```diff
@@ -0,0 +1,113 @@
+---
+{
+  title: "Chess Knight Problem: a quick and dirty solution in JavaScript",
+  description: "Here I present a quick and dirty solution to a common interview question where the solution is not nearly as complex as it may first appear.",
+  published: "2020-04-29T12:27:06.284Z",
+  authors: ["thodges314"],
+  tags: ["javascript", "chess", "knight", "computer science", "interview"],
+  attached: [],
+  license: "cc-by-4",
+}
+---
+
+# Chess Knight Problem: a quick and dirty solution in JavaScript
```

Comment:
Please remove this header. It's already present with the `title` frontmatter. You end up with duplicates otherwise:

![Two headings titles](https://user-images.githubusercontent.com/9100169/80640960-002d3680-8a19-11ea-8343-eab87edc65c3.png)